docs/conf.py

```diff
@@ -100,3 +100,22 @@
     "issue": ("https://github.com/fedora-infra/noggin/issues/%s", "#"),
     "pr": ("https://github.com/fedora-infra/noggin/pull/%s", "PR#"),
 }
+
+# -- Misc -----
+
+
+def run_apidoc(_):
+    from sphinx.ext import apidoc
+
+    apidoc.main(
+        [
+            "-f",
+            "-o",
+            os.path.join(topdir, "docs", "_source"),
+            os.path.join(topdir, "noggin"),
```

Comment:
We can safely exclude the tests, just add `noggin/tests` to the arguments list.

Reply:
not sure what you mean by this comment, can you clarify for me?

Reply:
Sure. As it is now, you are generating the apidocs for the unit tests module, and I don't think it's useful. If you add `noggin/tests` to the command line, it'll exclude them from apidoc generation.

Reply:
thanks for that!

fixed and updated!